config/environments/production.rb

```diff
@@ -20,9 +20,7 @@
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
   # config.require_master_key = true
 
-  # Disable serving static files from the `/public` folder by default since
-  # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  config.public_file_server.enabled = true
```

Comment:
hm, is this wise?  nginx can serve much faster without going through the rails stack.  Of course I think we only have a few images so it's not a big deal

Reply:
Currently we aren't using nginx (not too many users, not too many images).  If we add nginx later, we can put this flag back to false.